tests/Gedmo/Mapping/MappingEventAdapterTest.php

```diff
@@ -53,10 +53,10 @@ public function testAdapterBehavior(): void
             ->disableOriginalConstructor()
             ->getMock();
         $eventArgsMock->expects(static::once())
```

Comment:
Maybe we could even replace these mocks to use a class that is no deprecated, like `Doctrine\ORM\Event\PrePersistEventArgs` or `Doctrine\ORM\Event\PreUpdateEventArgs`.